### DISCUSSION TITLE
Turn off update notifications

### DISCUSSION THREAD

**Discussion by mgjgid** [nextcloud/all-in-one#698] [Questions] [answered]
Hello Everyone

is there a way to disable notifications for container updates?
(As we have multiple instances running, we get bombed with notifications about the updates).

**Comment by szaimen** [comment, marked as answer]
No, there is now way to turn them off. Simply do the upgrade and you are done or enable daily backups which will also update the containers automatically (and thus not show update notifications anymore).
